rpm/harbour-cameraplus.spec

```diff
@@ -62,13 +62,21 @@ cp %SOURCE2 $RPM_BUILD_ROOT/usr/share/icons/hicolor/86x86/apps/
 # qtcamera configuration
 mkdir -p $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/
 mkdir -p $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/jolla/
+ln -s jolla $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/JP-1301
```

Comment:
I don't get the point. The app ships with its configs so people will get the new app and new configs